tokens/deprecated/base.json

```
@@ -5,987 +5,1143 @@
         "value": "#ffefee",
         "type": "color",
         "deprecated": true,
-        "deprecatedComment": "replace cinnamon100 with {base.palette.red25}"
+        "deprecatedComment": "replace cinnamon100 with {base.palette.red25}",
+        "fallback": "{base.palette.red25}"
```

```suggestion
        "deprecatedComment": "replace cinnamon100 with {base.palette.red.25}",
        "fallback": "{base.palette.red.25}"
```
All refs should be with dots as object nested notation